cmake/Modules/FindMKLDNN.cmake

```diff
@@ -43,9 +43,9 @@ ELSE(MKL_FOUND)
 ENDIF(MKL_FOUND)
 
 SET(MKL_cmake_included TRUE)
-IF (NOT MKLDNN_THREADING)
-  SET(MKLDNN_THREADING "OMP:COMP" CACHE STRING "")
-ELSEIF (MKLDNN_THREADING STREQUAL "TBB")
+IF (NOT MKLDNN_CPU_RUNTIME)
```

Comment:
I think there are other places where this flag is references, including docs: https://github.com/pytorch/pytorch/search?q=MKLDNN_THREADING&unscoped_q=MKLDNN_THREADING

You can try to rename it, but given that it's about the same name - why not to keep original name?

Reply:
Thanks for pointing it out. Due it's a change inside DNNL to align with a newly added build option called `DNNL_GPU_RUNTIME`, so we have to use the new name. I have filed another patch to fix this.